### PR TITLE
Remove all mentions of TypeTuple from std.meta

### PR DESCRIPTION
**Pull request by MetaLang**

### CODE REVIEW

File: std/meta.d [hunk 1]
```diff
@@ -401,7 +401,7 @@ if (args.length >= 1)
 
 
 /**
- * Returns a typetuple created from TList with the all occurrences,
+ * Returns a `AliasSeq` created from TList with the all occurrences,
```
Comment (wilzbach) [inline std/meta.d]: I think this should have "an" as well (ditto for the other).

Comment (MetaLang) [inline std/meta.d]: Whoops, fixed.

**Comment by MetaLang** [comment]
I'm using the web UI so I can't at the moment. I'll do it in a bit.

**Comment by wilzbach** [comment]
> I'm using the web UI so I can't at the moment. I'll do it in a bit.

Ah I was just asking whether you agree -> squashed :)

**Pull request merged**